Include/object.h

```diff
@@ -756,7 +756,11 @@ PyType_HasFeature(PyTypeObject *type, unsigned long feature)
     // PyTypeObject is opaque in the limited C API
     flags = PyType_GetFlags(type);
 #else
-    flags = type->tp_flags;
+#   ifdef Py_GIL_DISABLED
+        flags = _Py_atomic_load_ulong_relaxed(&type->tp_flags);
```

Comment:
Can we use the `FT_ATOMIC_LOAD_ULONG_RELAXED` macro here?

Reply:
Not easily at least. This is the object.h file which is the "root" of all header files, so Id prefer we keep it out.

Reply:
Got it, thanks for the explanation!